src/Model/BlogMemberExtension.php

```diff
@@ -35,6 +35,13 @@ class BlogMemberExtension extends DataExtension
         'BlogProfileImage' => Image::class
     ];
 
+    /**
+     * @var array
+     */
+    private static $owns = [
```

Comment:
```suggestion
    private static array $owns = [
```
No need to use PHPDoc when we're declaring new properties - we can just directly use typehints.

For now the typehint doesn't actually enforce anything, but in the future the config might actually validate types against typehints, so we try to add them in whenever we're adding new config.

Reply:
Ah, good to know. Thanks. I was just making sure it was consistent with how it was done in `BlogPost.php`. https://github.com/silverstripe/silverstripe-blog/blob/37d2ccff76d04d6fd8c1f73146350a5e074d2652/src/Model/BlogPost.php#L88

Reply:
Thanks @purplespider! Yeah, there's sometimes a conflict between how it's done currently vs how it would be done ideally going forward.

I reckon in this case Guy could've just pushed the change, leave a comment explaining the same as above, and already have it merged ;-)

Reply:
I could have done. There's no hard and fast rules about what to let through and what not to. On the one hand it doesn't ultimately affect anything *right now* but on the other it's a small change so easy enough to make before merging.
I erred on the side of asking for the change in this case 🤷‍♀️